packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -484,6 +486,26 @@ export class StripeHandler {
     const sourceCountry =
```

Comment:
I know you didn't add this here, but we already have `paymentMethodCountry` above -- is `sourceCountry` different? I'm wondering if we need to execute this line at all. `paymentMethodCountry` is just block scoped, but that could be changed.

Reply:
Since the `paymentMethodId` arg can be empty, we need this.  I suppose an extra thing we could do is to check if `paymentMethodCountry` already exist and have a value before we call a function to get the country.  I'll keep that in mind in case I'm in this neighborhood again, but I'm not going to make that change in this PR.